zadanie_23.sql

```diff
@@ -0,0 +1,48 @@
+-- Tworzy tabelę pracownik(imie, nazwisko, wyplata, data urodzenia, stanowisko). W tabeli mogą być dodatkowe kolumny, które uznasz za niezbędne.
+CREATE TABLE pracownik(
+id BIGINT primary key auto_increment,
+first_name VARCHAR(30),
+last_name VARCHAR(30),
+salary INTEGER,
+birth_year DATE,
+title VARCHAR(30));
+-- Wstawia do tabeli co najmniej 6 pracowników
+INSERT INTO pracownik(first_name, last_name, salary, birth_year, title)
+VALUES('Jan', 'Kowalski', 5500, '1995-03-12', 'projektant'), ('Janina', 'Kowalska', 4500, '1998-09-19', 'asystent projektanta'),
+('Marta', 'Zawadzka', 8500, '1985-06-12', 'kierownik'), ('Jan', 'Nowak', 6500, '1990-10-03', 'asystent kierownika'),
+('Maria', 'Jankowska', 12500, '1975-03-18', 'dyrektor'), ('Paweł', 'Abacki', 6000, '1993-01-26', 'koordynator');
+-- Pobiera wszystkich pracowników i wyświetla ich w kolejności alfabetycznej po nazwisku
+SELECT*FROM zadanie_23.pracownik ORDER BY last_name;
+-- Pobiera pracowników na wybranym stanowisku
+SELECT*FROM zadanie_23.pracownik WHERE title = 'dyrektor';
+-- Pobiera pracowników, którzy mają co najmniej 30 lat
+SELECT*FROM zadanie_23.pracownik WHERE DATEDIFF(NOW(), birth_year)>=30*365.25;
```

Comment:
(-) Mnożenie przez 365.25 nie jest najlepszym pomysłem - nie zawsze to zwróci dobry wynik. Spróbuj wykorzystać funkcję EXTRACT.